lib/provider/map_notifier.dart

```diff
@@ -29,6 +31,20 @@ class MapNotifier with ChangeNotifier {
     notifyListeners();
   }
 
+  void setCampusLatLng(LatLng latLng) {
```

Comment:
- As long as the user isn't within Loyola, they are within SGW. Is that the desired result?

Reply:
Yes by default the value of the toggle remains sgw as long as the user does not enter loyola by going to it via the map or clicking the toggle to go to it